packages/typescript-client/test/integration.test.ts

```diff
@@ -548,7 +548,7 @@ describe(`HTTP Sync`, () => {
       { title: `foo8` }
     )
     // Wait for server to get all the messages.
-    await sleep(40)
+    await sleep(100)
```

Comment:
```suggestion
    await vi.waitFor(async () => {
      const res = await fetch(
        `${BASE_URL}/v1/shape/${issuesTableUrl}?offset=-1`
      )
      const body = (await res.json()) as Message[]
      expect(body.length).greaterThan(1)
    })
```

We use this approach elsewhere, probably better than fiddling with timeouts

Reply:
I think this is a good idea but maybe not for this PR.